### ISSUE TITLE
[feature] Option to stop/continue current Job with parallel tasks

### ISSUE THREAD

**Issue by ppcololo** [runabol/tork#264]
Will be cool if you implement some feature to stop current job.
Example: we have a job with 1000 parallel tasks. And in some moment we want to reconfigure workers - we dont want to cancel job and restart it - because sone of parallel tasks already compleated. 

If we have option `Stop` - tasks in job could change state to `Not active` for example and after we reconfigure workers or ENV or something else we can proceed Job with pressing `Continue`.

**Comment by runabol** [comment]
You should be able to terminate workers mid-processing. RabbitMQ will automatically re-queue the tasks and redistribute them to whatever workers are left online.

**Comment by ppcololo** [comment]
I know, but let's image that in one task I have some x200 sub tasks - like downloading\uploading something and I dont want to terminate this. I just want to let those tasks finish and the rest become onhold

**Comment by runabol** [comment]
They will finish. It's not going to restart the parent task. Each of the subtasks will get auto requeued. Those that finish will not be impacted.

**Comment by ppcololo** [comment]
Ok, let me rephrase:
I want to download 1m urls > I split this job to 1000 parallel tasks > and inside each task I have 1k urls
I have 100 workers. During the run of each task I have finished, lets image, 500 tasks in this Job. If I stop this Job all tasks will be marked as `Canceled` and if I `Restart` this Job - Tork will run all tasks that already `Completed` - 500 tasks already completed and I don't want to re-run them. 
So thats why I dont want to re-run completed tasks inside Job with parallel tasks.

**Comment by runabol** [comment]
Simply don't stop/cancel the job. Let it run. Terminate whatever machines you like, any tasks on these machines will be requeued to be handled by other workers.

**Comment by runabol** [comment]
So long as RabbitMQ stays online you should be fine with doing whatever you want with the workers and even the coordinator
